### PR TITLE
python 3.9.0

### PR DESCRIPTION
**Pull request by fxcoudert**
First try with Python 3.9.0 release candidate 1! https://www.python.org/downloads/release/python-390rc1/

### CODE REVIEW

File: Formula/python@3.9.rb [hunk 1]
```diff
@@ -0,0 +1,335 @@
+class PythonAT39 < Formula
+  desc "Interpreted, interactive, object-oriented programming language"
+  homepage "https://www.python.org/"
+  url "https://www.python.org/ftp/python/3.9.0/Python-3.9.0rc1.tar.xz"
+  sha256 "fba7a73e7359356967b853c234b97f4a4114f02b8bd60d6bde6e2f97bcdfdc70"
+  license "Python-2.0"
+
+  livecheck do
+    url "https://www.python.org/ftp/python/"
+    regex(%r{href=.*?v?(3\.8(?:\.\d+)*)/?["' >]}i)
```
Comment (dtrodrigues) [inline Formula/python@3.9.rb:10]:
```suggestion
    regex(%r{href=.*?v?(3\.9(?:\.\d+)*)/?["' >]}i)
```

File: Formula/python@3.9.rb [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+    url "https://files.pythonhosted.org/packages/08/25/f204a6138dade2f6757b4ae99bc3994aac28a5602c97ddb2a35e0e22fbc4/pip-20.1.1.tar.gz"
+    sha256 "27f8dc29387dd83249e06e681ce087e6061826582198a425085e0bf4c1cf3a55"
+  end
+
+  resource "wheel" do
+    url "https://files.pythonhosted.org/packages/75/28/521c6dc7fef23a68368efefdcd682f5b3d1d58c2b90b06dc1d0b805b51ae/wheel-0.34.2.tar.gz"
+    sha256 "8788e9155fe14f54164c1b9eb0a319d98ef02c160725587ad60f14ddc57b6f96"
+  end
+
+  # Remove this block when upstream adds arm64 compatibility
```
Comment (SMillerDev) [inline Formula/python@3.9.rb:59]: I'd assume this can go now?

File: Formula/python@3.9.rb [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+    system "make"
+
+    ENV.deparallelize do
+      # Tell Python not to install into /Applications (default for framework builds)
+      system "make", "install", "PYTHONAPPSDIR=#{prefix}"
+      system "make", "frameworkinstallextras", "PYTHONAPPSDIR=#{pkgshare}"
+    end
+
+    # Any .app get a " 3" attached, so it does not conflict with python 2.x.
+    Dir.glob("#{prefix}/*.app") { |app| mv app, app.sub(/\.app$/, " 3.app") }
```
Comment (SMillerDev) [inline Formula/python@3.9.rb:140]: Is this still needed?

File: Formula/python@3.9.rb [hunk 4]
```diff
@@ -0,0 +1,335 @@
+class PythonAT39 < Formula
+  desc "Interpreted, interactive, object-oriented programming language"
+  homepage "https://www.python.org/"
+  url "https://www.python.org/ftp/python/3.9.0/Python-3.9.0rc1.tar.xz"
```
Comment (cclauss) [inline Formula/python@3.9.rb:4]: Is it worth trying release candidate 2 or is that more work than it is worth?
* https://www.python.org/download/pre-releases/

Comment (dtrodrigues) [reply Formula/python@3.9.rb:4]: At this point yeah we can switch to rc2 since all that's really being tested is the build of python itself.

**Comment by cclauss** [comment]
https://www.python.org/downloads/release/python-390/

File: Formula/python@3.9.rb (hunk 3 above)
Comment (fxcoudert) [reply Formula/python@3.9.rb:140]: It's still in the main formula, so let's be consistent

**Comment by fxcoudert** [comment]
Building the 3.9.0 finale release!

**Comment by cclauss** [comment]
;-) refresh...

**Review by cclauss — APPROVED** [review]

**Review by dtrodrigues — APPROVED** [review]
🚀

**Pull request closed without merging**